libpod/runtime.go

```diff
@@ -233,3 +251,30 @@ func (r *Runtime) Shutdown(force bool) error {
 	// TODO: Should always call this even if store.Shutdown failed
 	return r.state.Close()
 }
+
+// Reconfigures the runtime after a reboot
+// Refreshes the state, recreating temporary files
+// Does not check validity as the runtime is not valid until after this has run
+// TODO: there's a potential race here, where multiple libpods could be in this
+// function before the runtime ready file is created
+// This probably doesn't matter as the actual container operations are locked
+func (r *Runtime) refresh(alivePath string) error {
```

Comment:
I'm just wondering if we need a refresh and a refreshAll function?  refreshAll would be this function called at anticipated init/restart time, then a refresh function that would take a container id as a parm and just refresh that one container.

I think your race condition concerns are valid and should be explored a bit.